src/plugins/intel_cpu/src/nodes/scaled_attn.h

```diff
@@ -8,6 +8,8 @@
 #include "node.h"
 #include "transformations/cpu_opset/common/op/sdpa.hpp"
 #include "utils/plain_tensor.hpp"
+#include "arm_compute/runtime/NEON/NEFunctions.h"
+#include "arm_compute/core/Types.h"
```

Comment:
Shouldn't these includes be in the corresponding cpp file and under `OV_CPU_WITH_ACL` definition check, to prevent any issues when the plugin is compiled without ACL?